### ISSUE TITLE
Need an updated popcorn-complete.js 

### ISSUE THREAD

**Issue by menismu** [menismu/popcorn-js#58]
Issue reported: [https://github.com/mozilla/popcorn-js/issues/467](https://github.com/mozilla/popcorn-js/issues/467)
Reported by: @steeevio

@steeevio
steeevio commented on 18 Nov 2015
The latest version on the website is 1.5.6 but I think the youtube api fixes were put in after that. Is it possible to get version 1.5.11 somewhere? Or how can I go about building my own? Simply including the relevant plugin files and wrappers in my site is not working.

@jyxia
jyxia commented on 4 Jan 2016
There is Makefile in this repo, so you can make it and build you own.

@NathanLawrence
NathanLawrence commented on 27 Apr 2016
I agree with @jyxia - this should be made as needed to make sure it's not out of sync with the main codebase, not added to the repository.

@tinyengines
tinyengines commented on 1 Feb 2017
Why? The project is dead - there should at least be a popcorn-complete.js and popcorn-complete.min.js for all those people who were linking to the now deprecated online versions.

**Comment by titaniumbones** [comment]
I agree with @tinyengines -- we should provide this.  should be easy to add, do you want to do this? It only means that before committing to master, `make build` will have to be run.

**Comment by tinyengines** [comment]
I would absolutely love to - but I just don't have the time these days to
contribute!  Definitely wish you all the success and the glory - I'll be
your first supporter!


*CONFIDENTIALITY NOTICE *: The contents of this email are confidential in
nature and are intended for the sole use of the intended recipient. If you
are not the intended recipient of this email, you have received this email
in error. Any disclosure, distribution, copying or action taken in reliance
on it is prohibited and may be unlawful.

**Comment by titaniumbones** [comment]
@menismu  any rewsponse? only asking b/c I want to include popcorn libraries in an assignment next week -- I can use my own, but I'd love to link to official docs if various PR's can be merged. Any chance of that?

**Comment by tinyengines** [comment]
doubt it - was a mozilla project, that they deprecated years ago - if I'm
not mistaken.


*CONFIDENTIALITY NOTICE *: The contents of this email are confidential in
nature and are intended for the sole use of the intended recipient. If you
are not the intended recipient of this email, you have received this email
in error. Any disclosure, distribution, copying or action taken in reliance
on it is prohibited and may be unlawful.

**Comment by titaniumbones** [comment]
Yes, bu @menismu has taken over maintainership and has been working on it for the last year

**Comment by titaniumbones** [comment]
Probably developers should add a pre-commit hook that runs `make build`. cf https://github.com/ddddavidmartin/Pre-commit-hooks Will implement and add something to README when i have a chance.

**Comment by vsoch** [comment]
Any progress here? I'm looking for that same file - adding the plugins individually didn't work.

**Comment by titaniumbones** [comment]
I have a version of popcorn-complete.js that I can give you privately -- unfortunately it is not really ready for publication.  I think @menismu has run out of time for this project, and I haven't been able to step up to maintain it myself properly. The code is getting a bit hard to read (lots of jQuery-style self-executing functions) and I'm not sure it's worth the effort to maintain it in its current form. Am definitely open to collaboration on updating if you're interested and have time/skills.  I'd really like to reimplement hte lugins as say stencil components but I am not at the point where I'm ready to do that.  

Hope that helps a little.
